Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,15 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import "./Header.css";
+import HeaderTitle from "./HeaderTitle";
+import ImageThumbnail from "./ImageThumbnail";
+
+const HeaderContent = () => {
+  return (
+    <div className="Header-content">
+      <ImageThumbnail />
+      <HeaderTitle />
```

Comment:
Good, but this should be in `HeaderContainer` with the other part I mentioned and then combine it all in `HeaderContainer`